report/regrade/classes/privacy/provider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+// This file is part of mod_offlinequiz for Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace offlinequiz_regrade\privacy;
+
+defined('MOODLE_INTERNAL') || die();
+
+class provider implements
+// This plugin does not store any personal user data.
+\core_privacy\local\metadata\null_provider {
+    /**
+     * This plugin does not store any data
+     *
+     * @return  string
+     */
+    public static function get_reason() : string {
+        return 'privacy:metadata';
```

Comment:
does this need a lang string added here? https://github.com/catalyst/moodle-mod_offlinequiz/blob/MOODLE_401_STABLE_fix/issue-247/report/regrade/lang/en/offlinequiz_regrade.php

or it pulls correctly from the parent?

Reply:
actually this string doesn't exist in the parent lang file either so will need to be added
https://github.com/catalyst/moodle-mod_offlinequiz/blob/MOODLE_401_STABLE_fix/issue-247/lang/en/offlinequiz.php

Reply:
Hey Jwalit, this language has now been added in report/regrade